docs/assets/css/copy-button.css

```diff
@@ -0,0 +1,27 @@
+.copy-button {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 5px 10px;
+    font-size: 14px;
+    cursor: pointer;
+    border: 1px solid #009E7F;

```

Comment:
Is this color present in the brand kit?
If no, then use Keppel for the button bg color and Carribean green for the hover effect.
https://layer5.io/company/brand

Reply:
![image](https://github.com/layer5io/getnighthawk/assets/64972958/9f03d07b-3c8a-4fa6-911a-35189ce0c736)
I have used these two colors from the design file of getnighthawk 
@Savio629